capstone/src/main/java/com/google/sps/data/Driver.java

```diff
@@ -6,15 +6,16 @@
   private final String times;
   private long seats;
   private final String email;
+  private final String license;
   private final String pollingAddress;
 
-  public Driver(
-      String first, String day, String times, long seats, String email, String pollingAddress) {
+  public Driver(String first, String day, String times, long seats, String email, String license, String pollingAddress) {
```

Comment:
add a newline here after "(" so it fits within 100 columns, or just newline after licence or whatever makes it not too long.